modules/host/contractmanager/storagefolder.go

```diff
@@ -90,6 +90,11 @@ type storageFolder struct {
 	atomicSuccessfulReads  uint64
 	atomicSuccessfulWrites uint64
 
+	// Atomic bool indicating whether or not the storage folder is available. If
+	// the storage folder is not available, it will still be loaded but return
+	// an error if it is queried.
+	atomicUnavailable uint64 // uint64 for alignment
```

Comment:
the size of a field doesn't impact its alignment. This field is 8-byte aligned because all the previous fields are 8 bytes.